config/database.yml

```diff
@@ -1,25 +1,16 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  adapter: postgresql
+  encoding: unicode
+  host: localhost
```

Comment:
why fixing this? Shouldnt this be gotten from the env var? What if I use a different username/password/pool etc?

Reply:
Yes but it's a common practice to not even track it in the version control. We usually have a `database.yml.example` in the repo, and every new dev creates it own private one. 
Thoughts?
```
"Everyone working on a project should be able to have it’s own database config."
```
http://yukas.by/rails/tips/2015/09/23/remove-database-yml-from-your-rails-repository/

Reply:
About 

```
In addition to that, I tried to set those values through the rails credentials but there is a problem that the rails core team is not considering. The issue is that when running rake db:create on development it creates both dev and test databases and when creating the test database it uses the dev credentials instead of the test credentials. That's another reason why I left it this way, I prefer to not solve this problem on the bolilerplate.
```

We should definitely solve this issue. Db credentials for test/dev should be the same, except for the database name.

Reply:
Just incorporated **_dotenv-rails_** as a workaround for this issue.